Library/Homebrew/os/mac/mach.rb

```diff
@@ -164,11 +164,10 @@ def initialize(path)
     end
 
     def parse_otool_L_output
-      ENV["HOMEBREW_MACH_O_FILE"] = path.expand_path.to_s
```

Comment:
I'm 100% sure this is used to escape path. See https://github.com/Homebrew/homebrew/blob/8f9854f81f8064c1428a05efc8d9754e92e259a7/Library/Homebrew/mach.rb#L153